portfolio/src/main/webapp/script.js

```diff
@@ -62,4 +59,25 @@ function addProjectDescription(project) {
   
   wrapper.innerText = details[project]; 
   wrapper.style.backgroundColor = colors[project];
+}
+
+/**
+ * This function enables the highlighting of a selected section from the navigation bar. 
+ * This function is called by the clicking of any one of the boxes in the navigation bar.
+ * Input: section to select
+ * Effect: selected section will have their border shown, and all the other sections will not have the border. 
+ */
+function highlightSection(select) {
+
+  document.getElementById(select).style.border = 'solid';
+  document.getElementById(select).style.borderColor = '#c5cf0a';
+
+  var navbarSections = document.getElementsByClassName('navbarSection');
+
+  for (var i = 0; i < navbarSections.length; i++) {
```

Comment:
I think rather than this plain for-loop you can either use `navbarSections.forEach` like:
```
navbarSections.forEach(
  section => {
    ...
  });
```

or a for-of loop like:
```
for (section of navbarSections) {
  ...
}
```

I think either of them is better than a plain for-loop because you don't have to do the bookkeeping with the index, but I prefer the first option more than the second. What do you think?

Reply:
I see! I agree that not having to worry about indexing is beneficial. I have changed the function into the forEach loop, with some necessary type conversion before.